src/js/models.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+var $ = window.$;
+var _ = window._;
+var Backbone = window.Backbone;
+
+var RegionDetailsModel = Backbone.Model.extend({
+    defaults: {
+        title: null,
+        description: null,
+        mapUrl: null,
+        image: null,
+        subregions: null,
+    },
+});
+
+window.AppModel = Backbone.Model.extend({
+    defaults: {
+        detailsVisible: false,
+        selectedRegion: null,
+        initialMapCenter: null,
+        initialMapZoom: null,
+        mapTilesMinZoom: null,
+        mapTilesMaxZoom: null,
+        mapTilesUrl: null,
+        mapTilesAttribution: null,
+        regionList: null,
+    },
+
+    initialize: function() {
+        var configData = null;
+
+        $.ajax({
```

Comment:
Backbone has a built in way to hydrate models, that doesn't require you to make a direct call to `$.ajax` or set the model properties directly. See http://backbonejs.org/#Model-fetch